src/main/java/org/folio/service/dataimport/utils/DataImportUtils.java

```diff
@@ -33,4 +39,36 @@ public static Map<String, String> extractOkapiHeaders(DataImportEventPayload eve
     }
     return headers;
   }
+
+  public static Map<String, String> kafkaHeadersToMap(KafkaConsumerRecord<String, String> kafkaRecord) {
+    var headerStream = kafkaRecord.headers().stream();
+    if (!isSystemUserEnabled()) {
```

Comment:
if I’m not confused then here we should use condition (isSystemUserEnabled()) (without `!` operator), correct? 
Because otherwise we will get headers without token when system user is disabled.

Reply:
thanks!